content/03.results.md

```diff
@@ -166,11 +166,11 @@ Frequent mutations in additional rare brain tumor histologies are depicted in **
 
 We analyzed mutational co-occurrence across the OpenPBTA, using a single tumor from each patient with available WGS (N = 666 patients). <!--SAMPLECOUNT-->
 The top 50 mutated genes (see **STAR Methods** for details) in primary tumors are shown in **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**).
-As expected, _TP53_ was the most frequently mutated gene across the OpenPBTA (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4), with all of these driven by HGGs and consistent with previous reports [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1093/neuonc/noaa251; @doi:10.1038/ng.2938].
+As expected, _TP53_ was the most frequently mutated gene across the OpenPBTA (8.7%, 58/668), significantly co-occurring with _H3F3A_ (OR = 30.05, 95% CI: 14.5 - 62.3, q = 2.34e-16), _ATRX_ (OR = 23.3, 95% CI: 9.6 - 56.3, q = 8.72e-9), _NF1_ (OR = 8.26, 95% CI: 3.5 - 19.4, q = 7.40e-5), and _EGFR_ (OR = 17.5, 95% CI: 4.8 - 63.9, q = 2e-4), with all of these driven by HGGs and consistent with previous reports [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1093/neuonc/noaa251; @doi:10.1038/ng.2938].
 
-In medulloblastoma tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3), as previously reported [@doi:10.1200/JCO.2010.31.1670], as well as with mutations in _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3), another commonly mutated gene in medulloblastomas [@doi:10.1038/nrc3410].
-Mutations in _FGFR1_ and _PIK3CA_ significantly co-occurred in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; @doi:10.1186/s40478-020-01027-z].
-Of HGG tumors with mutations in _TP53_ or _PPM1D_, 52/54 (96.3%) had mutations in only one of these genes (OR = 0.188, 95% CI: 0.04 - 0.94, p = 0.0413, q = 0.0587).
+In embryonal tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 43.6 95% CI: 7.1 - 265.8, q = 1.52e-3), as previously reported for medulloblastomas [@doi:10.1200/JCO.2010.31.1670], as well as with mutations in _DDX3X_ (OR = 21.4, 95% CI: 4.7 - 97.9, q = 4.15e-3), another commonly mutated gene in medulloblastomas [@doi:10.1038/nrc3410].
```

Comment:
```suggestion
In embryonal tumors, mutations in _CTNNB1_ significantly co-occurred with mutations in _TP53_ (OR = 43.6 95% CI: 7.1 - 265.8, q = 1.52e-3) as well as with mutations in _DDX3X_ (OR = 21.4, 95% CI: 4.7 - 97.9, q = 4.15e-3).
These events were driven by medulloblastomas and have been previously reported as significantly mutated in this tumor type [@doi:10.1038/nrc3410; 10.1200/JCO.2010.31.1670].
```
How about this?

Reply:
I like it!

Reply:
possibly I messed up that DOI piece...maybe no space?

Reply:
Noting this is being fixed in the PR where I'm adding in Table S2 to the manuscript.